rsconnect/bundle.py

```diff
@@ -270,6 +276,8 @@ def make_notebook_html_bundle(
         "--to=html",
         filename,
     ]
+    if noinput:
+        cmd.append('--no-input')    
```

Comment:
I notice that the nbconvert option is named `--no-input` and ours is `--noinput`. Should we match the nbcvonert name exactly?

Reply:
Also, if we're going to offer the option to hide tagged input cells we would need to include that here as well.